### PR TITLE
Add Symbol Colors.

### PR DESCRIPTION
**Pull request by ghost**
Modified version of icons.js.

### CODE REVIEW

**Comment by ghost** [comment]
Factions.js showing error instead of symbolcolor.js.

**Comment by DeathlyPlays** [comment]
Looks good, but then again I'm just now waking up. Will test and merge if successful. Thanks for doing this. Also, I'm gonna ignore the factions/typescript error on rooms.js and merge if this didn't add anything to the failed eslint.

Edit: Yup, its the typescript that failed it, don't worry.

File: server-plugins/symbolcolor.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+			updateSC();
+			this.sendReply("You removed " + target + "'s sc.");
+			Monitor.adminlog(user.name + " removed " + target + "'s sc.");
+			this.privateModCommand("(" + target + "'s symbol color was removed by " + user.name + ".)");
+			if (Users(target) && Users(target).connected) Users(target).popup("|html|" + Server.nameColor(user.name, true) + " has removed your symbol color.");
+		},
+	},
+	 schelp: [
+		"Commands Include:",
+		"/sc set [user], [color] - Gives [user] an symbol color of color]",
```
Comment (DeathlyPlays) [inline server-plugins/symbolcolor.js]: Period not ]

**Comment by ghost** [comment]
I've rewrite the help command. I've tested it on my server and it's working fine.

**Pull request merged**